docs/source/intro/installation.rst

```diff
@@ -391,19 +394,24 @@ To leave or deactivate the environment, simply run:
 .. admonition:: Update install tools
    :class: tip title-icon-tip
 
-   You may need to install ``pip`` and ``setuptools`` in your virtual environment in case the system or user version of these tools is old
+   You may need to install ``pip`` and ``setuptools`` in your virtual environment, in case the system or user version of these tools is old.
```

Comment:
```suggestion
   You may need to update ``pip`` and ``setuptools`` in your virtual environment, in case the system or user version of these tools is old.
```
They should always be installed by default no? And so at most they should just be updated. Or is them being installed automatically in a new venv just a feature of the `virtualenvwrapper` tool that I use?